roaring/fuzzer.go

```diff
@@ -24,3 +33,291 @@ func FuzzBitmapUnmarshalBinary(data []byte) int {
 	}
 	return 1
 }
+
```

Comment:
We might want to have a variant of this that we could run in non-fuzz testing, also?

But this looks reasonable to me. It's certainly a big improvement over what we had before.